applications/hpf/gpio/README.rst

```diff
@@ -1,16 +1,17 @@
-.. _sdp_gpio:
+.. _hpf_gpio:
 
-SDP GPIO
+HPF GPIO
 ########
 
-This application demonstrates how to create SDP on FLPR in Zephyr threadless mode by implementing a subset of Zephyr GPIO API.
+This application demonstrates how to create High Performance Framework on FLPR in Zephyr threadless
+mode by implementing a subset of Zephyr GPIO API.
```

Comment:
```suggestion
This application demonstrates how to create a high-performance framework on FLPR in Zephyr's threadless mode by implementing a subset of the Zephyr GPIO API.
```

Reply:
need to remove sdp_gpio refs from the following files: 
- release-notes-2.9.0.rst
- release-notes-2.9.0-nRF54H20-1.rst
this should fix the build